src/compiler/cpp_generator.cc

```diff
@@ -993,7 +995,23 @@ void PrintHeaderServerMethodCallback(
         "controller) {\n"
         "               return this->$"
         "Method$(context, request, response, controller);\n"
-        "             }));\n");
+        "             }, nullptr));\n}\n");
+    printer->Print(
+        *vars,
+        "void SetMessageAllocatorFor_$Method$(\n"
```

Comment:
I'm ok with leaving this as is and not part of some experimental substructure since the structure itself is marked experimental.